BlackNectar/BlackNectar/Fonts.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Fonts.swift
+//  BlackNectar
+//
+//  Created by Cordero Hernandez on 2/8/17.
+//  Copyright © 2017 BlackSource. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class Fonts {
+    
+    static let instance = Fonts()
+    private init() {}
+    
+    var oxygenBold = UIFont(name: "Oxygen-Bold", size: 16)
```

Comment:
Wonderful!
You could probably skip the whole singleton pattern here and just have these be static constant variables.
`Fonts.oxygenBold`, etc